tests/installation/ipxe_install.pm

```diff
@@ -285,6 +313,12 @@ sub run {
 
     select_console 'sol', await_console => 0;
 
+    if (is_sle('>=16')) {
+        record_info('Start agama installation');
+        assert_screen('agama-installer-live-root', 400);
+        return;
+    }
```

Comment:
Whole section is too early, you are completely skipping use of `set_bootscript_hdd` which is bellow.
Also do we need another needle tag, just for agama?  Its just installation.   I would just do condition for agama on line 338 and return.